tests/integ/test_auto_ml.py

```diff
@@ -62,6 +62,11 @@
 }
 
 
+@pytest.fixture(scope="module", autouse=True)
+def create_auto_ml_job_if_not_exist(sagemaker_session):
```

Comment:
what test is using this fixture?

Reply:
This fixture is autouse. But now a second thought made me realized not all tests in the class are using this fixture, I will apply it to individual tests.